eng/ci/templates/official/jobs/build-core-tools-host.yml

```diff
@@ -0,0 +1,57 @@
+jobs:
+- job: BuildCoreToolsHostWindows
+  displayName: '[Windows] Build CoreToolsHost'
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  steps:
+  - pwsh: |
+      Import-Module "./pipelineUtilities.psm1" -Force
+      Install-Dotnet
```

Comment:
Use the `UseDotnet` task

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-dotnet-v2?view=azure-pipelines

Reply:
done